simulator/modules/core/perf_instr.h

```diff
@@ -24,6 +24,7 @@ class PerfInstr : public FuncInstr
     bool is_bypassible() const {
         return !this->is_conditional_move()
             && !this->is_partial_load()
+            && !this->is_partial_store()
```

Comment:
There is no need to make stores bypassible or not as they write data to memory and not to registers.